service_api/grabbing_api/resources.py

```diff
@@ -49,65 +48,32 @@ class LatestDataResource(Resource):
-                if not characteristics["request_key"]:
-                    continue
-
-                model = characteristics["model"]
-                service_param = characteristics["request_key"]
-
-                model = getattr(models, model)
-                if not model:
-                    raise Warning(f"There is no such model named {model}")
-
-                if param in realty:
-                    obj = session.query(model).get(realty[param])
-                    if obj is None:
-                        raise BadRequestException("No such filters!")
-                    params[service_param] = obj.original_id
-        return params
-
     def post(self):
         """
         Returns latest information about realty and save it to DB
         """
 
         post_body = request.get_json()
-
-        try:
-            characteristics = post_body["characteristics"]
-            realty = post_body["realty_filters"]
-            additional = post_body["additional"]
-        except KeyError as error:
-            raise BadRequestException(error.args) from error
-
         metadata = open_metadata(PATH_TO_METADATA)
 
         for service_name in metadata:
             service_metadata = metadata[service_name]
-            params = self.convert_named_filed(realty, service_metadata)
-
-            type_mapper = process_characteristics(service_metadata, realty, CACHED_CHARACTERISTICS_EXPIRE_TIME,
-                                                  CACHED_CHARACTERISTICS)
-            params.update(dict((type_mapper.get(key, key), {"name": key, "values": value})
-                               for key, value in characteristics.items()))
-            params["page"] = (additional["page"] // additional["page_ads_number"]) + 1
-
-            items = RealtyRequesterToServiceResource().get(params, service_metadata)
-
-            try:
-                return process_request(items, dict(additional).pop("page"), additional.pop("page_ads_number"),
-                                       service_metadata)
-            except KeyError as error:
-                print(error.args)
-                raise BadRequestException(error.args) from error
+            request_to_domria = DomriaServiceHandler(post_body, service_metadata)
+            return request_to_domria.get_latest_data()
+
+
+            # url, params = DomRiaInputConverter(post_body, service_metadata).convert()
+            # response = requests.get(url=url, params=params, headers={'User-Agent': 'Mozilla/5.0'})
+            #
+            # # if response.status_code == 200:
+            # items = response.json()
+            #
+            # try:
+            #     return process_request(items, post_body["additional"].pop("page"),
+            #                            post_body["additional"].pop("page_ads_number"), service_metadata)
+            # except KeyError as error:
+            #     print(error.args)
+            #     raise BadRequestException(error.args) from error
```

Comment:
Can you remove this, please? Or If it is necessery uncomment it)

Reply:
Yes, sorry, forgot about this)